### PR TITLE
Link and unlink plane, crew and location to flight

### PR DESCRIPTION
**Pull request by jishnu28**
The functionalities of linking and unlinking the specified classes to flight work. 

However, I noted some minor issues regarding the way these links are displayed in the flight list and regarding link commands that attempt to link multiple instances to a flight within the same command. 

Will create new issues for these, to address them in v1.3. Will also include more details there!

### CODE REVIEW

**Comment by wxxedu** [comment]
Nice work! Will have a look once the style issues are fixed.

**Comment by tangyuantyy** [comment]
We can discuss about how we want the links are displayed in the flight list and the maximum number of instances linked to a flight within one command in the next meeting!

**Comment by tangyuantyy** [comment]
Other than that, the code looks good!

**Comment by jishnu28** [comment]
> We can discuss about how we want the links are displayed in the flight list and the maximum number of instances linked to a flight within one command in the next meeting!

yep sounds good!

File: src/main/java/seedu/address/logic/crew/linkcrew/LinkCrewCommand.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                        entry.getValue().getName()))
+                .collect(Collectors.joining(","));
+        return String.format(DISPLAY_MESSAGE, result, flight.getCode());
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        try {
+            for (Map.Entry<FlightCrewType, Crew> entry : crews.entrySet()) {
+                flight.crewLink.putRevolve(entry.getKey(), entry.getValue());
```
Comment (wxxedu) [inline src/main/java/seedu/address/logic/crew/linkcrew/LinkCrewCommand.java:63]: Need to think a bit about whether if the `putRevolve` is appropriate here though.

Comment (wxxedu) [reply src/main/java/seedu/address/logic/crew/linkcrew/LinkCrewCommand.java:63]: I will create a new issue here, should be something related to better error handling etc, and I guess we can merge for now.

File: src/main/java/seedu/address/storage/json/adapted/JsonAdaptedFlight.java [hunk 2]
```diff
@@ -128,34 +102,34 @@ public Flight toModelType() throws IllegalValueException {
...
         try {
-            Link<FlightPilotType, Pilot, ReadOnlyItemManager<Pilot>> link =
+            Link<FlightPilotType, Pilot, ReadOnlyItemManager<Pilot>> linkPilot =
                     new Link<>(Pilot.SHAPE, pilotLink,
                             GetUtil
                                     .getLazy(Model.class)
                                     .map(Model::getPilotManager)
                     );
-            flight = new Flight(id, code, link);
+            Link<FlightCrewType, Crew, ReadOnlyItemManager<Crew>> linkCrew =
```
Comment (wxxedu) [inline src/main/java/seedu/address/storage/json/adapted/JsonAdaptedFlight.java:111]: Ahhhh this looks much cleaner than before.

**Review by wxxedu — APPROVED** [review]

**Pull request merged**